reactor-netty-http/src/main/java/reactor/netty/http/server/HttpPredicate.java

```diff
@@ -36,7 +36,7 @@
  *
  * @author Stephane Maldini
  */
-final class HttpPredicate
+public final class HttpPredicate
```

Comment:
Not sure we can open publically that class

Reply:
I make it  package-private again.